docs-2.0/8.service-tuning/load-balance.md

```diff
@@ -43,27 +43,20 @@ After you add new storage hosts into the cluster, no partition is deployed on th
     +------------+
     | New Job Id |
     +------------+
-    | 2          |
+    | 25         |
```

Comment:
![image](https://user-images.githubusercontent.com/78209557/216501759-29496fc0-f817-405d-8e93-e21dffdbd40d.png)
It looks weird